readme.md

```diff
@@ -233,6 +233,16 @@
 | ------: | -------: | ------: |
 |   3 - 6 |       15 |     30m |
 
+### [One Night Werewolf](https://beziergames.com/products/one-night-ultimate-werewolf)
+
+> One Night Ultimate Werewolf is a fast-paced game where everyone gets to be a different role. In the course of only one night and the following morning, the players will determine who among them is a werewolf...hopefully.
+
+![werewolf game image](https://cdn.shopify.com/s/files/1/0740/4855/products/ONUW_3D_Box_250x.png?v=1461752063)
+
+| Players | Min. Age | Time    |
+| ------: | -------: | ------: |
+|   3 - 10|        8 |  10-15m |
```

Comment:
The linter looks to have flagged this line due to the lack of padding in the first cell.

```suggestion
|   3 - 10 |       8 |  10-15m |
```